### ISSUE TITLE
About how to use rotation from anygrasp，and why use points_y.

### ISSUE THREAD

**Issue by BJHYZJ** [ok-robot/ok-robot#11]
Thank you very much for such a good open source work. I have a question. Does the rotation obtained by anygrasp need to be multiplied by a matrix before it can be used? And why points_y at [link](https://github.com/ok-robot/ok-robot/blob/174c742b6a1866a2da8f75a89e2c544bb77ab5f9/ok-robot-manipulation/src/anygrasp_manipulation/object_handler.py#L321) needs to be negative?

**Comment by peiqi-liu** [comment]
Yes, the original anygrap gripper pose is grounded in coordinate frame specified by robot head camera. However, during manipulation every action is grounded in coordinate frame specified by the robot gripper, so rotation is needed. We have already included multiplication of rotation matrix between different coordinate frames in our codes.

Besides the rotation matrix between different coordinate frames, we also noticed that the images taken by robot realsense head camera are rotated by 90 degrees. So when computing depth we also needed some transformations to make sure computed pointcloud (x, y, z) is grounded in correct world frame. `-points_y` is one step of this transformation
